pmoves/tools/_secrets_common.py

```
@@ -122,19 +122,88 @@ def local_env_path() -> Path:
+
+    Checks for:
+    - Embedded newlines (literal \\n or actual newlines)
+    - Special characters that break Docker Compose parsing (+ in unquoted values)
+    - Suspicious patterns (base64 fragments concatenated to API keys)
+    """
+    if not value:
+        return True, None
+
+    # Check for literal \n (backslash-n) which indicates concatenation
+    if "\\n" in value:
+        return False, f"contains literal '\\\\n' - value appears to be multiple values concatenated"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)